prow/cmd/mkbuild-cluster/README.md

```diff
@@ -55,6 +55,14 @@ Note: restart plank to see the updated values.
 
 ## More options:
 
+### Credential errors
+
+By default we validate the new client works before printing out its credentials.
+
+The `--get-client-cert` flag may fix these errors.
```

Comment:
What is the behavior with and without this? I thought `mkbuild-cluster` only output cert-based admin credentials to clusters?

Reply:
Yes, it will only output those. But a) they may not work and b) without it, if you're using something else it will just fail.

Currently the error message is:
```
FATA[0000] Failed: authenticated client could not list pods: response has status "403 Forbidden" and body "{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"pods is forbidden: User \"client\" cannot list pods in the namespace \"kube-system\"","reason":"Forbidden","details":{"kind":"pods"},"code":403}
```

Which does not make it super obvious that this means "we are trying to validate these credentials work, and it looks like they don't"